stable/gocd/templates/NOTES.txt

```diff
@@ -1,52 +1,37 @@
 1. Get the GoCD server URL by running these commands:
 {{- if .Values.server.ingress.enabled }}
     It may take a few minutes before the IP is available to access the GoCD server.
-    ip=$(kubectl get ingress --namespace {{ .Release.Namespace }} {{ template "gocd.fullname" . }}-server -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
-    echo "http://$ip"
+        $ echo "GoCD server public IP: http://$(kubectl get ingress {{ template "gocd.fullname" . }}-server --namespace={{ .Release.Namespace }}  -o jsonpath='{.status.loadBalancer.ingress[0].ip}')"
 {{- range .Values.server.ingress.hosts }}
-  http://{{ . }}
+    http://{{ . }}
 {{- end }}
 {{- else if contains "NodePort" .Values.server.service.type }}
-    export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "gocd.fullname" . }}-server)
-    export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
-    echo http://$NODE_IP:$NODE_PORT
+    $ export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "gocd.fullname" . }}-server)
+    $ export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+    $ echo http://$NODE_IP:$NODE_PORT
```

Comment:
Putting a `$` in front breaks copy/pasting multiple lines.

Reply:
Done. Removed `$` from all the lines.